### ISSUE TITLE
resources_dir_path() should provide an absolute path

### ISSUE THREAD

**Issue by zmike** [servo/servo#5986]
The current implementation requires that the exe be 2 subdirs into the servo/ root directory, eg. servo/a/b, in order to function. Instead, this should be an absolute path defined at compile time.

**Comment by jdm** [comment]
How would that work? Wouldn't that limit the resulting binary to being run only on the machine that compiled it, in the original directory?

**Comment by zmike** [comment]
No, you allow the path to be defined based on configure time so that the user can determine where the resources directory will be. There can be a default path when the user doesn't specify anything.
